shared/actions/chat.js

```diff
@@ -499,17 +546,25 @@ function * _deleteMessage (action: DeleteMessage): SagaGenerator<any, any> {
 
   if (messageID) {
     // Deleting a server message.
-    // TODO: Use delete message RPC call when it is available
     const clientHeader = yield call(_clientHeader, CommonMessageType.delete, conversationIDKey)
-    clientHeader.supersedes = messageID
+    const conversationState = yield select(_conversationStateSelector, conversationIDKey)
```

Comment:
since i'm in here and actually send some malformed delete i'm updating this as its very close to the edit